### ISSUE TITLE
No IPV6 Support?

### ISSUE THREAD

**Issue by D3nnis3n** [mfreiholz/ts3video#59]

**Comment by jedi404** [comment]
ipv6 support shouldnt be expected just yet. most ISP's in the US dont even
fully offer ipv6 networking yet. not to mention. i doubt ts3video would
really need to be running ipv6 in the first place.

**Comment by D3nnis3n** [comment]
Please don't get me wrong, but i don't care about the US.
In Germany IPV6 is pretty functional - every user of our teamspeak has native IPV6.

As our teamspeak is reached through a dns name that uses ipv6 first and ipv4 as fallback there is no possibility for us to join the conference server without knowing the exact ipv4 of the teamspeak server. This is pretty bad and i would consider this critical. (As in: Not able to use, if ipv4 is not known to the users - and they don't want to know it, they want it easy. It's easy through dns-name. And i really will not disable ipv6 on my server.)

Anyways, it was just a suggestion / bug report and not an expectation. But if you directly ask me if i would expect ipv6 compatibility i would definatley say yes and fully disagree with you.

**Comment by jedi404** [comment]
i can understand your situation but its not just the US that doesnt have
fully functional ipv6. i just happen to live in the US and there are alot
of members using it in the us. now i understand the dev is german too so
things will be different on your internet than my own but from my view its
not a priority. once again i can understand its different in your country.
thats fine. in the mean time might i suggest an ipv6 to ipv4 tunnel?

**Comment by D3nnis3n** [comment]
It's fine if it is not a priority - but i really don't see the sense of you commenting against priority for IPV6. To get IPV6 spread out it is needed that people adapt their systems. It's not that hard. Also, since TeamSpeak 3.1 (and this software has been updated for this version already), IPV6 is offically supported and recommended by TeamSpeak. I really can't agree with your opinion. I'm using IPV6 for ages without problems. It's available widespread in Germany and there are no reasons to not support the future despite the dev doing this in his freetime (thanks for that!). Other countries not being able to adapt to IPV6 is the problem of the ISPs in those countrys and not mine, the one of german ISPs, users or developers. It's your ISPs that are bad, not the need for ipv6, ipv6 itself or using ipv6. I would yell at my ISP (and in fact did some years ago!) if i wouldn't have IPV6. But i would never argue against it, because it is not usable for me. :O

**Comment by jedi404** [comment]
sorry i have to laugh at this. but only because with he other conversation
we have going theres a reason i said ipv6 isnt priority. lol weve got
issues with cameras, and in  my groups case not being able to run the
client on any monitor but the main one or it just fails entirely.
personally id be prioritizing bugs over new features but thats just me.
that being said though. the US is hardly a candidate for good internet.
last i checked the ISP's are all being greedy tightwads and not even
letting us regular joes have 100Mbps in some areas. even though its clear
our network s in the states could handle it. id guees the point on that is
that the reason i dont see ipv6 as important is because its still so new
here vs by the sound of things its been around forever in germany. all in
all though id assume its something that will eventually be added to the
application as networks continue to evolve and the internet gets bigger.
but thats just me looking at the future objectively.

**Comment by D3nnis3n** [comment]
Well, i see, but you forget something: I consider missing IPV6 a bug (This is not a feature, this is a must-have!). As we can't use the tool with it it's also a critical one - as we use ipv6 nativley for years now. It's not around here for ages nativeley, but nearly everyone at least has IPV6 through provider-tunneling and is therefore able to use it. Well, internet here is dependant on where you live, too - but yes, i got 400 MBps and you can get that over cable nearly everywhere. But there are still areas where you can't get over 6 Mbps or even less. In european comparision Germany is also very bad at internet speed - so i'm really asking myself what is wrong in your country.

This discussion will lead to nothing, anyways, so just hope the dev will fix both issues when he has got time *g*

**Comment by jedi404** [comment]
yes i agree the conversation didnt go very far toward actually fixing the
issue but i enjoy being able to have a friendly discussion with another
user. also as far as whats wrong in my country its all just greed but thats
a topic for another day. overall id say yer right though, even though i
cant see the use in ipv6 you need it to function properly which does indeed
make that a bug rather than a feature. at least as far as your concerned.
for people like me weve already been over it does absolutely nothing for.
that being said on a personal level good luck with this issue, and
hopefully you can resolve it or find a workaround. other than that stay
safe out there, and have a good one sir.
